internal/cli/runner_profile_list.go

```diff
@@ -35,18 +36,32 @@ func (c *RunnerProfileListCommand) Run(args []string) int {
 
 	c.ui.Output("Runner profiles")
 
-	tbl := terminal.NewTable("Name", "Plugin Type", "OCI Url", "Default")
+	tbl := terminal.NewTable("Name", "Plugin Type", "OCI Url", "Target Runner",
+		"Default")
 
 	for _, p := range resp.Configs {
 		def := ""
 		if p.Default {
 			def = "yes"
 		}
 
+		var targetRunner string
+		if p.TargetRunner != nil {
+			switch t := p.TargetRunner.Target.(type) {
+			case *pb.Ref_Runner_Any:
+				targetRunner = "*"
+			case *pb.Ref_Runner_Id:
+				targetRunner = t.Id.Id
+			}
+		} else {
+			targetRunner = "*"
```

Comment:
I like this notation, but what happens if someone sets their runner profile name to `"*"`? 🤔 I know we have this issue else where in the CLI, i.e. how to convey "everything" versus a name

Reply:
Hahaha was trying to sneak this one by. Yeah, I thought about that too... but figured it's obscure enough of an edge case... Do you know what the resolution is?

Reply:
In the case with run triggers targeting all applications in a project, I've just been leaving it empty string. I don't think empty string is a valid application, so it's probably not a valid runner profile either. That's as good as I got at the moment 😄

Reply:
I think we should have some way to differentiate between "any" and "nil" here. There have been multiple times when I've had an issue with runner profiles that have `nil` for their target, and it would be useful for me to differentiate that state from `any`.

I think the odds of someone naming their profile `*` are pretty low, and that's the best name I can come up with, but am open to other suggestions.

Reply:
Fair enough! The nil == empty string case makes more sense than the all == empty string case 👍🏻